exercises/robot-simulator/example.rs

```diff
@@ -0,0 +1,101 @@
+#[derive(PartialEq, Debug, Copy, Clone)]
+pub enum Direction {
+    North,
+    East,
+    South,
+    West,
+}
+
+impl Direction {
+    pub fn previous_clockwise(&self) -> Self {
+        match self {
```

Comment:
nit: These matches are generally written in the form: 
```rust
match *self {
    Direction::North => Direction::West,
    ...
}
```

(Note the lack of reference sigils in the patterns)